packages/wells/src/client/baseWellsClient.ts

```diff
@@ -148,3 +162,39 @@ export default class BaseWellsClient {
     return this.http;
   }
 }
+
+export class ConfigureAPI {
+  protected client?: HttpClientWithIntercept;
+  protected project?: String;
+  protected cluster?: String;
+
+  public set setHttpClient(httpClient: HttpClientWithIntercept) {
+    this.client = httpClient;
+  }
+
+  public set setProject(project: String) {
+    this.project = project;
+  }
+
+  public set setCluster(cluster: String) {
+    this.cluster = cluster;
+  }
+
+  protected getPath(baseUrl: string, cursor?: string): string {
+    if (this.project == undefined) {
+      throw new HttpError(400, 'The client project has not been set.', {});
+    }
+    if (this.cluster == undefined) {
+      throw new HttpError(400, 'No cluster has been set.', {});
+    }
+    let path =
+      this.cluster == Cluster.BLUEFIELD
+        ? `/${this.project}${baseUrl}`
+        : `/${this.project}${baseUrl}?env=${this.cluster}`;
```

Comment:
It is only the cogdev environment that needs an `?env` variable. And only if you use something other than `api`.
You could create a function that takes in a cluster and returns a pair of `baseUrl`, and `env`. And if `env` is `!= null`, then use that.